docs/user-guide/attributes-table.md

```diff
@@ -8,17 +8,23 @@ In [MapStore](https://mapstore.geo-solutions.it/mapstore/#/), through the <img s
 
 Accessing this panel the user can perform the following main operations:
 
-* Manage records
+* [Edit records](attributes-table.md#editing-and-removing-existing-features) through the <img src="../img/button/edit_button.jpg" class="ms-docbutton"/> button 
 
-* Set filters
+* Filter records in Attribute Table in different ways as described in the [Set filter](attributes-table.md#set-filters) section below
 
-* Download the grid data
+* Opening the [Advanced Search](filtering-layers.md#query-panel) tool through the <img src="../img/button/filter-layer.jpg" class="ms-docbutton"/> button
 
-* Customize the Attribute table display
+* Activating the filtering capabilities by [clicking on map](filtering-layers.md#quick-filter-by-map-interaction), through <img src="../img/button/filter_geometry_button.jpg" class="ms-docbutton"/> button 
 
-* Create [Widgets](widgets.md), with the <img src="../img/button/widgets.jpg" class="ms-docbutton"/> button
+* Using the [quick filter by attribute](filtering-layers.md#quick-filter-by-attributes)
 
-* Zoom to features, with the <img src="../img/button/zoom-feature.jpg" class="ms-docbutton"/> button
+* Download the grid data through the <img src="../img/button/download_attribute_table.jpg" class="ms-docbutton"/> button 
```

Comment:
Ths icon is out to date, use the new one. The same wrong icon image is user in many places of this section.